### PR TITLE
Move develop versions from KGS to opengever.core.

### PR DESCRIPTION
**Pull request by jone**
Move the versions from the KGS opengever/develop (version 0ca4f64) to `opengever.core`'s `versions.cfg`.
Copied 1:1 from https://github.com/4teamwork/kgs/blob/0ca4f64/release/opengever/develop
I will propose some cleanup separately.

As discussed in #3084
See also https://github.com/4teamwork/kgs/pull/111

### CODE REVIEW

**Comment by jone** [comment]
Ok, I have added more changes, including the celery pinnings.
Changes:
- add a `bin/cleanup-versions-cfg` script, which cleans up the `versions.cfg` (remove duplicates, sort, remove comments)
- document version management in readme
- cleanup versions
 -move `plone.formwidget.autocomplete` from sources to versions
- remove unnecessary ldap sources redefinition
- add celery version pinnings (for testing, because of  `opengever.pdfconverter`)
- add development / testing

**Comment by jone** [comment]
bildschirmfoto 2017-07-11 um 10 38 01

File: versions.cfg [hunk 1]
```diff
@@ -1,21 +1,214 @@
...
+ftw.mobilenavigation = 1.4.6
+ftw.pdfgenerator = 1.6.0
+ftw.profilehook = 1.2.1
+ftw.raven = 1.2.0
+ftw.recipe.deployment = 1.2.0
+ftw.recipe.translations = 1.2.6
+ftw.showroom = 1.3.1
+ftw.tabbedview = 3.8.2
+ftw.table = 1.18.1
+ftw.testbrowser =
```
Comment (deiferni) [inline versions.cfg]: i think we should also pin `ftw.testbrowser`. i fwe don't and testbrowser changes it is not guaranteed that we can checkout an old version and tests still pass.

Comment (jone) [inline versions.cfg]: @deiferni oh, missed that. Thanks! Updated.

**Comment by jone** [comment]
@phgross please merge whenever you're ready

**Pull request merged**